### ISSUE TITLE
sql: introduce <precision> and <scale> for numeric types

### ISSUE THREAD

**Issue by Korablev77** [tarantool/tarantool#3672]
According to ANSI, there are two types of numbers: exact numeric value or an approximate numeric value. The last one can feature \<precision> and \<scale>. For example: `DECIMAL(5, 2)` is a number in format \<xxx.xx>, where 5 is precision and 2 - scale.

To be more accurate, entire quote from ANSI:

> A numeric type is described by a numeric type descriptor. A numeric type descriptor contains:
> — The name of the specific numeric type (NUMERIC, DECIMAL, SMALLINT, INTEGER, BIGINT, FLOAT, REAL, or DOUBLE PRECISION).
> — The implemented precision of the numeric type.
> — For an exact numeric type, the implemented scale of the numeric type.
> — An indication of whether the precision (and scale) are expressed in decimal or binary terms.
> — The name by which the numeric type was declared (but the corresponding full name if INT or DEC was specified).
> — The explicit declared precision, if any, of the numeric type.
> — For an exact numeric type, the explicit declared scale, if any, of the numeric type.
> If <precision> or <scale> is not specified explicitly, then the corresponding element of the descriptor effectively contains the null value.

**Comment by zilveer** [comment]
Is this implemented yet ?

**Comment by Korablev77** [comment]
@zilveer No, it is not. Ticket now is in wishlist which indicates its extremely low priority.  Yet @kyukhin may re-consider this decision.
